conf/application.conf

```diff
@@ -22,12 +24,15 @@ play.filters.enabled += "play.filters.cors.CORSFilter"
 play.filters {
   csrf.header.bypassHeaders {
     X-Requested-With = "*"
+    Csrf-Token = "nocheck"
   }
   hosts {
-    allowed = ["rcruitme:9000", "localhost:9000"]
+    allowed = ["."]
```

Comment:
prob add these back later and add prod host. right now base url is just using load balancer url. gonna get a domain name.